src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -41,13 +51,20 @@ function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
             <TableRow>
               {columns.map(column => (
                 <TableCell key={column.id} className={classes.tableHead}>
-                  {column.label}
+                  <TableSortLabel
+                    classes={{ root: classes.sortLabel }}
+                    active={sortColumn === column.id}
+                    direction={sortColumn === column.id ? sortDirection : 'asc'}
+                    onClick={() => handleChangeOrderBy(column.id)}
```

Comment:
`column.id` is strongly typed here:
https://github.com/WFP-VAM/prism-frontend/blob/de8105cc82950b7006e39ab3ab7b8eef112eab9e/src/utils/analysis-utils.ts#L52
but we're losing it by classifying it to string. 
Reckon we can use `Column['id']` instead of `string`?

Reply:
changed to `Column['id']`